sdk/src/system_instruction.rs

```diff
@@ -65,6 +65,7 @@ pub enum SystemInstruction {
     /// * space - memory to allocate if greater then zero
     /// * program_id - the program id of the new account
     CreateAccountWithSeed {
+        base: Pubkey,
```

Comment:
@rob-solana would there be any harm in passing `base` in on the keys list instead of via instruction parameter?  I need to be able to trick both the transaction signing and address generation verification to do an offline stake account creation with seed.

cc/ @garious

Reply:
I don't understand how pulling base into keys list helps.  The base will be the offline pubkey..

Reply:
It helps because our current, hacky pubkey/signature replacement for submission of offline signed TXs can't do the swap from the serialized instruction data. Since the base sig is required and the pubkey is also in the ix data, but we only specify these with one parameter, we wind up in the irreconcilable situation where the submission side TX construction either has the wrong pubkey in ix data, or too many signers. Moving onto the keys list frees us from needing to specify the offline pubkey in the ix and the signature swap fixes everything up for us after construction.

I'll put a PR up in a few and we can continue there

Reply:
https://github.com/solana-labs/solana/pull/8252